app/views/layouts/_activity_history.html.erb

```diff
@@ -0,0 +1,24 @@
+<div class="table-custom-scrollbar my-2">
+  <table class="table table-bordered table-striped mb-0">
+    <thead>
+      <tr>
+        <th scope="col">#</th>
+        <th scope="col">Activity</th>
+        <th scope="col">Time</th>
+      </tr>
+    </thead>
+    <tbody>
+      <% @activities.each_with_index do |activity, index| %>
```

Comment:
if you need starting index from 1 then use each.with_index(1)

Reply:
done